haxepunk/HXP.hx

```diff
@@ -273,12 +244,7 @@ class HXP
 		if (value < 0) value = 0;
 		if (_volume == value) return value;
 		_volume = value;
-		#if flash
-		_soundTransform.volume = value;
-		SoundMixer.soundTransform = _soundTransform;
-		#else
 		Sfx.onGlobalUpdated(false);
-		#end
```

Comment:
Note for later, we can move all this sound-related stuff out of HXP, maybe into Sfx.